### ISSUE TITLE
this app can't run on your pc

### ISSUE THREAD

**Issue by pndtsonu** [scout-app/scout-app#270]
i am using 32bit Operating System on Windows 10. when i click on exe file to install, below pic showing error.

![image](https://cloud.githubusercontent.com/assets/20009722/19190538/bc6b084a-8cbb-11e6-9523-270a85246af6.png)

**Comment by TheJaredWilcurt** [comment]
@pndtsonu 

Thanks for reporting this. I'll take a look on my Win 10 VM when I get home. 

Just to help explain, no installation is required, you just need to be able to run `Scout-App.exe`. I know that Windows 10 added in "security" features as a way to force people to use their marketplace to buy more desktop apps through them so they get a cut rather than going directly to the source.

You may just need to change this "feautre" in your OS settings to allow for running of applications that did not come through the marketplace.

1. Open **Settings app** and navigate to **Update & Security** section.
2. Go to **For developers** tab and try either **Sideload apps** or **Developer mode**

![developer-mode](https://cloud.githubusercontent.com/assets/4629794/19192513/73fee8ca-8c74-11e6-8108-8b77255de286.jpg)

If that works for you, let me know. I'll investigate if it's worth the effort to get Scout-App in their marketplace.

**Comment by TheJaredWilcurt** [comment]
Looks like trying to get stuff to run on Windows 10 the "Microsoft Way" is a huge pain in the ass. Way too many hoops to jump through.

**Comment by pndtsonu** [comment]
@TheJaredWilcurt 
I have tried both mode but still getting same error.

**Comment by TheJaredWilcurt** [comment]
@pndtsonu 

I tried with Win 10 Pro 64-Bit (Sideload Apps enabled) and Scout-App 2.0.2 and 2.5.7 both work.

I'm creating a Win 10 Pro 32-Bit ISO right now and will create a VM for it to test with. Scout-App itself is a 32-bit application, so it should work on all Windows versions.

![screenshot](https://cloud.githubusercontent.com/assets/4629794/19237406/b233a43e-8eca-11e6-97a1-6d6a3b832425.png)

**Comment by TheJaredWilcurt** [comment]
@pndtsonu 

Ha! I figured it out, that took forever. I made a Win 10 32 VM and got the same error as you. I tried everything with the OS and couldn't get it to work. Blah blah blah, long story short, I found out that I had mistakenly distributed a 64-Bit version of Scout-App instead of a 32-Bit (which runs fine on 64-Bit).

This will be fixed in the next release. I'll notify you when it comes out.
